src/graphql/utils/formats/default.ts

```diff
@@ -0,0 +1,48 @@
+import { camelCase, pascalCase, constantCase } from 'change-case'
+import FormatName from '../../schema/FormatName'
+
+const formatInsideUnderscores = (formatter: (name: string) => string) => (fullName: string) => {
+  const [, start, name, finish] = /^(_*)(.*?)(_*)$/.exec(fullName)!
+  return `${start}${formatter(name)}${finish}`
+}
```

Comment:
This is just the current logic: https://github.com/calebmer/postgraphql/blob/7eea049eea6703909dcb99fd7771f4032d6f4d09/src/graphql/utils/formatName.ts